frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -359,16 +360,25 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.postselect = self.in_classical_tracers[-2]
+        self.reset = self.in_classical_tracers[-1]
+
     def trace_quantum(self, ctx, device, trace, qrp) -> QRegPromise:
         op = self
         wire = op.in_classical_tracers[0]
         qubit = qrp.extract([wire])[0]
-        postselect = op.in_classical_tracers[1]
-
-        qubit2 = op.bind_overwrite_classical_tracers(ctx, trace, qubit, postselect=postselect)
+        qubit2 = op.bind_overwrite_classical_tracers(ctx, trace, qubit)
+        # postselect = op.in_classical_tracers[1]
+        # qubit2 = op.bind_overwrite_classical_tracers(ctx, trace, qubit, postselect=postselect)
```

Comment:
```suggestion
        qubit2 = op.bind_overwrite_classical_tracers(ctx, trace, qubit)
        # TODO: execute post-selection depending on qnode config
        # qubit2 = op.bind_overwrite_classical_tracers(ctx, trace, qubit, postselect=op.postselect)
```

Reply:
Thanks. Should we do this in a follow-up PR?

Reply:
Yes that's fine, Mudit will be doing this in the PR that implements the config options :)